config/detekt.yml

```diff
@@ -85,10 +59,14 @@ complexity:
     active: false
   LargeClass:
     active: true
-    threshold: 150
+    # Had to increase the threshold as RC13 started counting lines of code
+    # see https://github.com/mozilla-mobile/fenix/issues/4861
+    threshold: 200
   LongMethod:
     active: true
-    threshold: 60
+    # Had to increase the threshold as RC13 started counting lines of code
+    # see https://github.com/mozilla-mobile/fenix/issues/4861
```

Comment:
Okay, will remove that.

Reply:
Done.

Reply:
P.S. Just wonder why :)

Reply:
@dector in components we try to not create a dependency with our consuming apps, so that we solve the bug for all android component-based browsers. For that reason, if there's a Fenix bug that needs to be fixed in a-c we usually file an a-c issue with the abstract problem and reference it in code with a link to the Fenix bug. 🙂

Reply:
Totally make sense for me